kernel/classes/ezcontentobjecttreenodeoperations.php

```diff
@@ -113,6 +113,7 @@ static function move( $nodeID, $newParentNodeID )
                 $nodeIDList = array( $nodeID );
                 eZSearch::removeNodeAssignment( $node->attribute( 'main_node_id' ), $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList );
                 eZSearch::addNodeAssignment( $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList, true );
+                eZSearch::refreshChildren( $node );
```

Comment:
@emodric / @peterkeung One issue found with just using delayed indexing as is, http cache will get out of date (purged before index is updated). We have similar issues when soft commit is set to a high number, but in this case it will be worse as it will be both soft commit time and interval of the cronjob which would typically be much longer that the soft commit time.

We might need to emit an indexing event here on index so new stack can purge the affected locations.